card/db/migrate_core_cards/data/more_bootstrap.json

```diff
@@ -0,0 +1,51 @@
+{
+  "url": "http://localhost:3000/export.json",
+  "timestamp": "2015-02-25 14:55:17 +0100",
+  "card": {
+    "name": "export",
+    "type": "Pointer",
+    "content": "[[style: glyphicons]]\n[[simple skin]]\n[[simple bootstrap skin]]\n[[Default Layout]]\n[[*all+*script]]\n[[*page header]]\n[[style: bootstrap compatible]]\n[[Old Layout]]",
+    "value": [
```

Comment:
I don't think there's anything on wagn.org.  Agree that we need docs. IMO this kind of documentation we should mostly write inline in the code and get serious about generating docs from the code like rails does.  And of course we'll want to refer to it online.

I think the format's pretty solid.  content = raw db content.  value we use for processing the content in this case (generating a list of items from that). Can be done recursively.  We may add some extra detail, but I think most of what's there is exactly as we need it.